Lua/Libraries/CYK/AttackManager.lua

```diff
@@ -341,7 +341,7 @@ return function(CYK)
             color = { 1, 1, 1 }
         end
 
-        CYK.UI.CreateHPChangeText(textValue, target, color)
+        CYK.UI.CreateChangeText(textValue, target, color)
```

Comment:
Why was the function's name changed here?

Reply:
Since it's now used for HPs AND Mercy Percentage, I changed its name to be more global.
It really was just changed because I'm a maniac-

Reply:
In that case, naming it CreateValueChangeText would be better I think.

Reply:
Got it, I'll change it